go.mod

```diff
@@ -4,9 +4,14 @@ require (
 	github.com/fxamacker/cbor/v2 v2.2.0
 	github.com/go-kit/kit v0.8.0
 	github.com/go-logfmt/logfmt v0.4.0 // indirect
+	github.com/go-playground/locales v0.13.0
+	github.com/go-playground/universal-translator v0.17.0
 	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/google/uuid v1.1.0
+	github.com/leodido/go-urn v1.1.0 // indirect
```

Comment:
@lindseysimple Be advised this dependency itself has a dependency on a project that is [no longer maintained](https://github.com/pmezard/go-difflib). Per recent discussions in the architecture meetings, this probably won't be acceptable.

https://github.com/leodido/go-urn/blob/a0f5013415294bb94553821ace21a1a74c0298cc/go.sum#L3

Are there alternatives?

Reply:
[leodido/go-urn](github.com/leodido/go-urn) is updated to v1.2.0 after [go-playground/validator](https://github.com/go-playground/validator) is updated to v10.2.0. This line has been removed after running `go mod tidy`.